lib/lsperfm/core/stats.rb

```diff
@@ -0,0 +1,64 @@
+# encoding: utf-8
+require "thread"
+
+Thread.abort_on_exception = true
+
+module LogStash::PerfM
```

Comment:
What is the M for? PerformanceMeter/Measure/.?

Maybe Benchmark or PerformanceTest? I suck at naming..